src/pydvl/influence/torch/influence_function_model.py

```diff
@@ -876,3 +880,321 @@ def to(self, device: torch.device):
         return ArnoldiInfluence(
             self.model.to(device), self.loss, self.low_rank_representation.to(device)
         )
+
+
+class EkfacInfluence(TorchInfluenceFunctionModel):
+    r"""
+    Solves the linear system Hx = b, where H is the Hessian of a model with the empirical
+    categorical cross entropy as loss function and b is the given right-hand side vector.
+    It employs the EK-FAC method [@george2018fast], which is based on the kronecker
+    factorization of the Hessian first introduced in [@martens2015optimizing].
+    Contrary to the other influence function methods, this implementation can only
+    be used for classification tasks with a cross entropy loss function. However, it
+    is much faster than the other methods and can be used efficiently for very large
+    datasets and models. For more information, see [Eigenvalue Corrected K-FAC][ekfac].
+
+    Args:
+        model: Instance of [torch.nn.Module][torch.nn.Module].
+        hessian_regularization: Regularization of the hessian.
+    """
```

Comment:
My suggestion would be to make all the methods related to the details of ekfac invisible in the documentation, i.e. add an underscore (make them "private").
If you really want to expose them to the user, my suggestion would be to make the docstrings complete, i.e. with input and output description in google format.